### ISSUE TITLE
Test case with fixture within class of cases doesn't load - fixture not found

### ISSUE THREAD

**Issue by jonastim** [smarie/python-pytest-cases#129]
I am struggling to make a test case inside a class of cases work with a fixture. 
In the same file there are other tests that load the fixture just fine. Loading the case as a free function in the file also works fine (`@parametrize_with_cases("input", cases=[case_fixture])` or `@parametrize_with_cases("input", cases=".")`. However, if I try to load a case with a fixture from any other file or from a class of cases it complains that it can't find the fixture with the name of the case function (minus the `case_` prefix).

Minimal example:
```
# @fixture - tried pytest_cases fixture and regular one
@pytest.fixture
def my_fixture():
    return "dummy-fixture"

class CaseClass:
    def case_fixture(self, my_fixture):
        return my_fixture

@parametrize_with_cases("input", cases=CaseClass)
def test_me(
    input: str
) -> None:
    assert input
```
```
test setup failed
file <makefun-gen-12>, line 1: source code not available
E       fixture 'fixture' not found
>       available fixtures: <...>, my_fixture, <...>
```

From this [doc](https://smarie.github.io/python-pytest-cases/#cases-requiring-fixtures) I suspect this should work. 
Tested with `pytest-cases 2.1.3`

**Comment by smarie** [comment]
Thanks @jonastim ! This has been fixed in 2.2.1:

```python
from pytest_cases import fixture, parametrize_with_cases

# @fixture - tried pytest_cases fixture and regular one
@fixture
def my_fixture():
    return "dummy-fixture"

class CaseClass:
    def case_fixture(self, my_fixture):
        return my_fixture

@parametrize_with_cases("input", cases=CaseClass)
def test_me(
    input: str
) -> None:
    assert input
```

works fine. Can you please confirm ?

**Comment by smarie** [comment]
Closing this issue as it has been fixed in 2.2.1. @jonastim please feel free to reopen if this is not the case
